components/grid/grid.html

```diff
@@ -0,0 +1,146 @@
+                </li>
+                <li class="sidebar-item">
+                    <a class="component-link" href="/components/input/input.html">Input</a>
+                </li>
+                <li class="sidebar-item">
+                    <a class="component-link" href="/components/list/list.html">List</a>
+                </li>
+                <li class="sidebar-item">
+                    <a class="component-link" href="/components/modal/modal.html">Modal</a>
+                </li>
+                <li class="sidebar-item">
+                    <a class="component-link" href="/components/rating/rating.html">Rating</a>
+                </li>
+                <li class="sidebar-item">
+                    <a class="component-link" href="/components/navigation/navigation.html">Navigation</a>
+                </li>
+                <li class="sidebar-item">
+                    <a class="component-link" href="/components/typography/typography.html">Typography</a>
+                </li>
+                <li class="sidebar-item">
+                    <a class="component-link" href="/components/toast/toast.html">Toast/Snackbar</a>
+                </li>
+                <li class="sidebar-item sidebar-item-active">
+                    <a class="component-link" href="/components/grid/grid.html">Grid</a>
+                </li>
+            </ul>
+            </aside>
+            <main class="main">
+                <div class="component_wrapper">
+                    <h2 class="component_heading">Grid</h2>
+                    <p>
+                        Grid is one of the most efficient CSS utilities to divide block of elements equally in 2D form. We can divide elements in how many parts as we want, but basic partition is <span class="component_description">2 items in grid</span> and <span class="component_description">3 items in grid</span>
+                    </p>
+                    <h3 class="component_heading">2 items in Grid</h3>
+                
+                    <div class="component_wrapper_box">
+                        <div class="grid grid-2">
+                            <div class="grid border-black">
+                                <img src="/assets/avatar_sample.jpg"/>
+                                <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+                                    Ducimus amet, adipisci sit placeat ipsam fugit quis ut ad illo
+                                    omnis at libero reiciendis deleniti. Autem sequi asperiores culpa aperiam et.</p>
+                            </div>
+                            <div class="grid border-black">
+                                <img src="/assets/avatar_sample.jpg"/>
+                                <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+                                    Ducimus amet, adipisci sit placeat ipsam fugit quis ut ad illo
```

Comment:
remove lorem ipsum text add some meaningful text as suggested in roc8 common pointer guide